x-pack/legacy/plugins/apm/mappings.json

```diff
@@ -61,9 +61,6 @@
       },
       "apm_oss.metricsIndices": {
         "type": "keyword"
-      },
-      "apm_oss.apmAgentConfigurationIndex": {
```

Comment:
@dgieselaar can you tell me why have you removed this?

Reply:
See last bullet item of the changes in the description:
> Removed agentConfigurationIndex from configurable UI indices. Should be created and queried with internalUser, so making this configurable could lead to security issues.